### PR TITLE
Removed the "Delete profiles" action from the web profiler sidebar

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | yes |
| Tests pass? | yes |
| Fixed tickets | #15819 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by fabpot** [comment]
LGTM. Do we add a command for the CLI to purge the profiles? Or is `cache:clear` enough (which does not purge profiles if one use a non-based filesystem profile storage)? ... which leads to my next question: do we really need several storages for profiles? Looks like keeping only the filesystem one is more than enough. Thoughts?

**Comment by javiereguiluz** [comment]
My opinion:

  * Don't provide any command (`cache:clear` is enough)
  * Remove all storages except the filesystem one

**Comment by fabpot** [comment]
@javiereguiluz Let's do it. Can you create a PR that deprecates all storages but the filesystem one? I think we can still keep the interface just in case.

**Comment by javiereguiluz** [comment]
Done in #15944.

**Comment by stof** [comment]
:+1:

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig [hunk 1]
```diff
@@ -486,32 +486,15 @@ tr.status-warning td {
     padding-bottom: 99999px; {# needed for 'same-height columns' trick #}
     width: 220px;
 }
-#sidebar a {
-    color: #ADD8E6;
-}
```
Comment (xabbuh) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: Don't we still need this for the log out link?

Comment (javiereguiluz) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: Nope. In the toolbar we use `.sf-toolbar-block .sf-toolbar-info-piece a`. I've tried to make the toolbar and the profiler as independent as possible.

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Pull request merged**